tests/Api/ServiceTest.php

```diff
@@ -80,6 +80,39 @@ function () { return []; }
         $this->assertArrayHasKey('foo', $s->getOperations());
     }
 
+    public function serviceIdDataProvider()
```

Comment:
Would be nice to have a set of cases in here with trailing numbers as well.

Reply:
Added a couple of cases with trailing numbers.